docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -278,6 +278,23 @@
     "estimator.options.resilience.pec.max_overhead = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a6c87cd7",
+   "metadata": {},
+   "source": [
+    "## When to use each method\n",
+    "\n",
+    "| Method | When to use | Notes|\n",
+    "| --- | --- | --- |\n",
+    "| Dynamical decoupling | Use with circuits in which one or more qubits have long idle times | Circuits with dense two-qubit gates structures (for example, see the paper [Evidence for the utility of quantum computing before fault tolerance](https://doi.org/10.1038/s41586-023-06096-3)). Also, Heron devices are less susceptible to the dominating noise that is suppressed by simple DD sequences (cross-talk) |\n",
```

Comment:
> Use with circuits in which one or more qubits have long idle times

How do I know if my qubits have long idle times? 

> Circuits with dense two-qubit gates structures (for example, see the paper [Evidence for the utility of quantum computing before fault tolerance](https://doi.org/10.1038/s41586-023-06096-3)).

What is this sentence trying to say? That I can find an example of a dense circuit in the paper? How do I know if my circuit is considered "dense" or not? 

In addition, what kind of DD sequences should I use?